LongWarOfTheChosen/Src/LW_Overhaul/Classes/X2LWAbilitiesModTemplate.uc

```diff
@@ -95,7 +95,16 @@ static function UpdateAbilities(X2AbilityTemplate Template, int Difficulty)
 			AddImmuneConditionToPoisonSpit(Template);
 			break;
 		case 'AdvPurifierFlamethrower':
-			AddImmuneConditionToFlamethrower(Template);
+			UpdatePurifierFlamethrower(Template);
+			break;
+		case 'PriestStasis':
+			MakePriestStasisNotEndTurn(Template);
+			break;
+		case 'HolyWarriorDeath':
+			RemoveTheDeathFromHolyWariorDeath(Template);
```

Comment:
"Warrior" :wink:

Reply:
In all seriousness, if someone does a search for "HolyWarrior" in all files in the project, it won't pick this up.

Reply:
Good Point